articles/evangelist-and-chameleon.md

```diff
@@ -0,0 +1,61 @@
+
+# Possibility 1: The Whiner
+
+Andrea will complain continuously about the tool she is forced to use. It is slow and a certain bug leads to undesired behaviour at least daily. And anyway, this tool is simply not as good as her favourite tool. Even after years, Andrea complains about the same problems. She has neither learnt how the tool works nor found any of the workarounds her teammates use nor taken any steps to replace the tool.
+
+The whiner tries to convince others that her preferences are the best, but she doesn't take any steps to actually switch to her preferred tool.
+
+The advantage of the whiner is short-lived: It is a good way to find out whether others dislike the same tool and (if you listen closely) whether there are any reasons why this tool might be the right one for the situation.
+
+The disadvantages are that whiners are annoying and there's the risk that people stop listening to Andrea even when discussing unrelated topics.
+
+# Possibility 2: The Evangelist
+
+Whoever wants to listen (and everyone else, too) will hear continuously about all the advantages of Andrea's favourite tool. She does everything she can to have as many people as possible switch to her favourite tool, even if there are good reasons why up to now, something else was in use.
+
+The big challenge of evangelists is that they usually don't know the alternatives. This means that Andrea sounds more like "I don't want to learn about this tool" than like "I know something better". In the worst case, the functionality she points out as the biggest advantage has been implemented in an improved way in the very alternative she wants to replace.
+
+The advantage of the evangelist is that many migrations need the tenacity of an evangelist. Without evangelists, we would still use the tools from the last century.
+
+The disadvantage is that Andrea might overlook a far more important issue or a precondition for the migration.
+
+# Possibility 3: The Tightrope Walker
+
+Where possible, Andrea uses her preferred tool. Whether she adheres to the team's conventions or not is a balancing act:
+
+ * If your environment differs too much from everybody else's, pair programming will become more difficult.
+ * Most companies have rules about what can be installed/used. Because Andrea most likely doesn't know the reasoning for each of these rules, she has to consider carefully which rules she breaks.
+ * A tool you know well and that is configured to fit your preferences can increase your productivity and reduce your frustration.
+
+The difference between a tightrope walker and an evangelist is that the tightrope walker focuses on her own productivity and comfort. She doesn't attempt to convince anybody, but simply uses whatever she prefers (as long as she believes that it is ok within the context).
+
+The advantage of the tightrope walker is that because the specific tools are less important, Andrea can look at the big picture and focus on the most important issues.
```

Comment:
Satz klingt komisch. Kann man den Nebensatz "because the specific tools are less important" anders einbauen? Oder in Klammern geben?